CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [v1.5.1](https://github.com/megaease/easegress/tree/v1.5.1) (2022-04-06)
+
+[Full Changelog](https://github.com/megaease/easegress/compare/v1.5.0...v1.5.1)
+
+**Significant changes:**
+
+- Turn profiling on/off runtime [\#543](https://github.com/megaease/easegress/pull/543)
+
+**Implemented enhancements:**
+
+- Change way StatusSyncController stores statuses [\#542](https://github.com/megaease/easegress/pull/542)
+- Support mesh monitor mTLS [\#528](https://github.com/megaease/easegress/pull/528)
+- Support custom image name [\#545](https://github.com/megaease/easegress/pull/545)
+- Remove package contexttool [\#547](https://github.com/megaease/easegress/pull/547)
+- Update go version to 1.17 [\#546](https://github.com/megaease/easegress/pull/546)
```

Comment:
`mesh monitor mTLS` is for EaseMesh;
`contexttool` and `go 1.17` are not user-visible changes.

```suggestion
- Change the way StatusSyncController stores statuses to reduce memory usage [\#542](https://github.com/megaease/easegress/pull/542)
- Support custom image name [\#545](https://github.com/megaease/easegress/pull/545)
```